src/Perspex.SceneGraph/Animation/CrossFade.cs

```diff
@@ -8,12 +8,29 @@
 
 namespace Perspex.Animation
 {
+    // TODO: Perhaps we want a common base for Transitions with Duration so that we can 
+    // consolidate the property declarations, etc
+    //
+
     /// <summary>
     /// Defines a cross-fade animation between two <see cref="IVisual"/>s.
     /// </summary>
-    public class CrossFade : IPageTransition
+    public class CrossFade : PerspexObject, IPageTransition
```

Comment:
The downside is that `PerspexObject` is a fair bit more heavyweight, so I'd only use it if something is a) styleable or b) needs to be bindable. I'm not sure if a transition would be either of those, but I wouldn't like to say for definite. I just wondered if you had a scenario in mind where it would need one of those things.